### DISCUSSION TITLE
Older style taskbar

### DISCUSSION THREAD

**Discussion by AAUCrisp** [Open-Shell/Open-Shell-Menu#1713] [Ideas]
I know the newest version of W11 starts supporting "not grouping" the stuff we have open on the taskbar, but I still miss being able to have a taskbar that is not meant for being "picked up" by the binoculars on my grandpa's nose... 

These are some of the features that has kept me from "updating" to W11, and when I finally gave W11 a chance on my laptop recently, I got confirmed that I hate it.

I could get around this issue using the StartAllBack program, but I prefer the configuration options and functionality of OpenShell.
OpenShell is better in every way... except for the lack of "my old taskbar".

No matter what the answer is... 
Thank you guys so much, for this amazing program!

**Comment by tallman69** [comment]
I use ExplorerPatcher https://github.com/valinet/ExplorerPatcher to get the Windows 10 taskbar back on Windows 11.  ExplorerPatcher has worked very well for me.  I especially like that it allows me to enable the Quick Launch toolbar and bring back the older style context menus.
